JSON/Fold.tmPreferences

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<plist version="1.0">
+<dict>
+    <key>scope</key>
+    <string>source.json</string>
+    <key>settings</key>
+    <dict>
+        <key>indentationFoldingEnabled</key>
+        <false/>
+        <key>foldScopes</key>
+        <array>
+            <dict>
+                <key>begin</key>
+                <string>punctuation.section.sequence.begin</string>
+                <key>end</key>
+                <string>punctuation.section.sequence.end</string>
+            </dict>
+            <dict>
+                <key>begin</key>
+                <string>punctuation.section.mapping.begin</string>
+                <key>end</key>
+                <string>punctuation.section.mapping.end</string>
+            </dict>
```

Comment:
Maybe of lower priority, but do we want to find a common scheme of sorting those rules?
```suggestion
            <dict>
                <key>begin</key>
                <string>punctuation.section.mapping.begin</string>
                <key>end</key>
                <string>punctuation.section.mapping.end</string>
            </dict>
            <dict>
                <key>begin</key>
                <string>punctuation.section.sequence.begin</string>
                <key>end</key>
                <string>punctuation.section.sequence.end</string>
            </dict>
```

Reply:
I don't think there's ever going to be enough for it to really matter.